rpcd/playbooks/roles/rpc_maas/files/plugins/disk_status.py

```diff
@@ -0,0 +1,165 @@
+#!/usr/bin/env python
```

Comment:
Tried this on an HP and got:

```
root@rpc-openstack:/usr/lib/rackspace-monitoring-agent/plugins# ./disk_status.py --raid-type hp
Traceback (most recent call last):
  File "./disk_status.py", line 145, in <module>
    hp_raid = hpssacli_check()
  File "./disk_status.py", line 81, in hpssacli_check
    'failing_disks': failing_disks,
NameError: global name 'failing_disks' is not defined
```

Don't have an Adaptec card to test on yet though or a Dell.